transient/sshfs.py

```diff
@@ -21,6 +26,11 @@
 
 _PROVISION_SCRIPTS = {"RHEL": _RHEL_PROVISION_SCRIPT, "FEDORA": _RHEL_PROVISION_SCRIPT}
 
+global provision_lock
+provision_lock = threading.Lock()
+global provision_success
```

Comment:
That is correct and it bugs me too. However, like `if ssh_proc.stderr is not None:`, mypy, and therefore `make check`, fail if I don't include those global lines even if they contribute nothing to the program.
Luckily, putting these variables into SshfsThread, like you suggested in another comment, allows us to avoid the issue.